assets/airports/kmia.json

```diff
@@ -42,495 +42,1258 @@
-                "ZFP": ["ZFP", "LUCSS", "FOWEE"],
-                "ZQA": ["ZQA", "TINKY", "LUCSS", "FOWEE"],
-                "HODGY": ["HODGY", "TINKY", "LUCSS", "FOWEE"],
-                "URSUS": ["URSUS", "FOWEE"]
+                "HODGY": ["HODGY", "TINKY", "LUCSS"],
+                "URSUS": ["URSUS"],
+                "ZFP": ["ZFP", "LUCSS"],
+                "ZQA": ["ZQA", "TINKY", "LUCSS"]
             },
-            "body": [["JUNUR", "A120|S280"], "LUVLY", "DHP"],
-            "draw": [["ZFP*","LUCSS","FOWEE"],["ZQA*","TINKY","LUCSS","FOWEE"],["HODGY*","TINKY","LUCSS","FOWEE"],["URSUS*","FOWEE"],["FOWEE","JUNUR","LUVLY","DHP"]]
+            "body": ["FOWEE", "JUNUR", "LUVLY", "DHP"],
+            "rwy": {
+                "KMIA8L": [],
+                "KMIA8R": [],
+                "KMIA9": [],
+                "KMIA12": [],
+                "KMIA26L": [],
+                "KMIA26R": [],
+                "KMIA27": [],
+                "KMIA30": []
+            },
+            "draw": [
+                ["HODGY*", "TINKY", "LUCSS", "FOWEE", "JUNUR", "LUVLY", "DHP"],
+                ["URSUS*", "FOWEE"],
+                ["ZFP*", "LUCSS", "FOWEE"],
+                ["ZQA*", "TINKY", "LUCSS"]
+            ]
         },
         "HILEY6": {
             "icao": "HILEY6",
             "name": "Hiley Six",
             "entryPoints": {
-                "ZFP": ["ZFP", ["STAPL", "A180"], ["MILSY", "A160|S280"], "BOYUR", "HILEY"],
-                "OMN": ["OMN", "MALET", "PCMAN", "MLB", "TRV", "PHORD", "SMUGS", "MORGA", ["PBI", "A170"], ["BTOGA", "A130|S280"], "HILEY"],
-                "JORAY": [["JORAY", "S280"], "OSOGY", "AMZAR", ["STAPL", "A180"], ["MILSY", "A160|S280"], "BOYUR", "HILEY"]
+                "JORAY": ["JORAY", "OSOGY", "AMZAR", "STAPL", "MILSY", "BOYUR"],
+                "OMN": ["OMN", "MALET", "PCMAN", "MLB", "TRV", "PHORD", "SMUGS", "MORGA", "PBI", "BTOGA"],
+                "ZFP": ["ZFP", "STAPL", "MILSY", "BOYUR"]
+            },
+            "body": ["HILEY", "KAINS", "CIMBA", "JESSS", "RUBOE", "#271"],
+            "rwy": {
+                "KMIA8L": [],
+                "KMIA8R": [],
+                "KMIA9": [],
+                "KMIA12": [],
+                "KMIA26L": [],
+                "KMIA26R": [],
+                "KMIA27": [],
+                "KMIA30": []
             },
-            "body": ["HILEY", "KAINS", "CIMBA", "JESSS", "RUBOE"],
-            "draw": [["OMN*","MALET","PCMAN","MLB","TRV","PHORD","SMUGS","MORGA","PBI","BTOGA","HILEY"], ["JORAY*","OSOGY","AMZAR","STAPL","MILSY","BOYUR","HILEY"], ["ZFP*","STAPL"], ["HILEY","KAINS","CIMBA","JESSS","RUBOE"]]
+            "draw": [
+                ["JORAY*", "OSOGY", "AMZAR", "STAPL", "MILSY", "BOYUR", "HILEY", "KAINS", "CIMBA", "JESSS", "RUBOE"],
+                ["OMN*", "MALET", "PCMAN", "MLB", "TRV", "PHORD", "SMUGS", "MORGA", "PBI", "BTOGA", "HILEY"],
+                ["ZFP*", "STAPL"]
+            ]
         },
         "SSCOT3": {
```

Comment:
SSCOT3 is now SSCOT4

Reply:
reverified and updated all SIDs/STARs.

done